common/dynamicconfig/constants.go

```diff
@@ -514,6 +514,11 @@ is currently processing a task.
 		4*1024,
 		`NexusEndpointExternalURLMaxLength is the maximum length of a Nexus endpoint external target URL.`,
 	)
+	NexusEndpointDescriptionMaxSize = NewNamespaceIntSetting(
```

Comment:
Slightly confusing that this is subject to a different size limit than the rest of our payload fields, but probably unlikely to come up in practice.

Reply:
The user metadata payloads also have different limits BTW.